src/v3/src/transformer/password/passwordSettingsUtils.ts

```diff
@@ -71,6 +68,11 @@ export const getComplexityItems = (complexity?: ComplexityRequirements): ListIte
   return items;
 };
 
+/**
+ *
+ * @deprecated This function is no longer used as v3 decided not to display
+ * server-side requirements in the list of password requirements.
+ */
```

Comment:
when will we be able to remove this code?

Reply:
We could probably remove it now since its essentially dead code. I just didn't remove it because I wasnt sure if we were going to eventually bring back server-side requirements. But if not, we can totally remove it now.

Reply:
I default towards removing dead code and if we want it back we use git history

Reply:
Good deal, I will get rid of this. 🙏